src/victory-transition/victory-transition.jsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import VictoryAnimation from "../victory-animation/victory-animation";
+import { Transitions } from "../victory-util/index";
+import defaults from "lodash/defaults";
+import pick from "lodash/pick";
+
+export default class VictoryTransition extends React.Component {
+  static propTypes = {
+    /**
+     * The animate prop specifies an animation config for the transition.
+     * This prop should be given as an object.
+     */
+    animate: React.PropTypes.object,
+    /**
+     * VictoryTransition animates a single child component
+     */
+    children: React.PropTypes.node,
+    /**
+     * This prop specifies which of the child's props are safe to interpolate.
+     * This props should be given as an array.
+     */
+    animationWhitelist: React.PropTypes.array
+  };
+
+  componentWillReceiveProps(nextProps) {
+    this.setState(this.getTransitionState(this.props, nextProps));
+  }
+
+  getTransitionState(props, nextProps) {
+    const { animate } = props;
+    if (!animate) {
+      return {};
+    } else if (animate.parentState) {
```

Comment:
Is `animate.parentState` necessary?  This seems harder to reason about than managing state from a single place.